### PR TITLE
Fix prefetch issue while priming

### PR DESCRIPTION
**Pull request by akong-sf**
Fixes issue where contacts are not being displayed offline

### CODE REVIEW

File: force-app/main/default/lwc/accountRelatedContacts/accountRelatedContacts.js [hunk 1]
```diff
@@ -14,8 +14,6 @@ export default class AccountRelatedContacts extends NavigationMixin(
     There is a workaround that can be implemented in this Knowledge Article {@link https://help.salesforce.com/s/articleView?language=en_US&id=000396405&type=1}.
   */
   get accountQuery() {
-    if (!this.recordId) return undefined;
```
Comment (Taost) [inline force-app/main/default/lwc/accountRelatedContacts/accountRelatedContacts.js:17]: The RelatedRecords document should be also updated as the previous graphql issue has been address in 248 orgs. https://github.com/salesforce/offline-app-developer-starter-kit/blob/main/docs/RelatedRecords.md

Comment (akong-sf) [reply force-app/main/default/lwc/accountRelatedContacts/accountRelatedContacts.js:17]: Good point. Will update it

**Review by dbreese — APPROVED** [review]
LGTM!

**Review by Taost — APPROVED** [review]

**Comment by akong-sf** [comment]
@dbreese I don't have write access to merge this PR. Can you please do it?

**Pull request merged**